pandas/core/window/rolling.py

```diff
@@ -568,6 +569,46 @@ def calc(x):
         else:
             return self._apply_tablewise(homogeneous_func, name)
 
+    def _numba_apply(
+        self,
+        func: Callable[..., Any],
+        numba_cache_key_str: str,
+        engine_kwargs: dict[str, bool] | None = None,
+        numba_args: tuple[Any, ...] = (),
+        **kwargs,
+    ):
+        window_indexer = self._get_window_indexer()
+        min_periods = (
+            self.min_periods
+            if self.min_periods is not None
+            else window_indexer.window_size
+        )
+        obj = self._create_data(self._selected_obj)
+        if self.axis == 1:
+            obj = obj.T
+        values = self._prep_values(obj.to_numpy())
+        if values.ndim == 1:
```

Comment:
this is fine i am not sure if we are consistent about this pattern or np.atleast_2d

Reply:
Yeah I've seen both in the codebase

Reply:
cc @jbrockmendel a future point of consistency. prob not a big deal though.

Reply:
we're not consistent, no